lib/matplotlib/figure.py

```diff
@@ -989,7 +990,30 @@ def legend(self, *args, **kwargs):
         no legend being drawn.
 
 
-        **2. Labeling existing plot elements**
+        **2. Explicitly defining the artists and labels in the legend**
```

Comment:
```suggestion
        **2. Explicitly listing the artists and labels in the legend**
```
defining seems like not the right word to me since the variables are defined when the artists are created